src/main/java/pl/javastart/task/Firma.java

```diff
@@ -1,9 +1,17 @@
 package pl.javastart.task;
 
+import pl.javastart.task.Operacje.Wydatki;
+import pl.javastart.task.Podatki.FormaOpodatkowania;
+import pl.javastart.task.Operacje.Przychody;
+import pl.javastart.task.Operacje.Transakcje;
+
 public class Firma {
 
+    private final int iloscOperacji = 1000;
     private String nazwa;
+    private final Transakcje[] transakcje = new Transakcje[iloscOperacji];
```

Comment:
Gdy wprowadzisz jedną klasę Transakcja, to tutaj stwórz dwie osobne tablice na przychody i wydatki